src/utils/word.ts

```diff
@@ -1,74 +1,110 @@
-import fs from 'fs';
+import fs from "fs";
+import path from "path";
 
-const wordList: string[] = fs.readFileSync('src/sgb-words.txt')
-    .toString()
-    .split(/\r?\n|\r/);
+const wordList: string[] = fs
+  .readFileSync("data/sgb-words.txt")
+  .toString()
+  .split(/\r?\n|\r/);
```

Comment:
most of these changes were applied by my auto formatter, which should (?) follow the eslint rules set up in this repo. Happy to revert if they don't match, though.